wallet-standard/dapp/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "solana-dapp-next",
+  "version": "0.1.0",
+  "license": "MIT",
+  "private": false,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@heroicons/react": "^1.0.5",
+    "@identity.com/cryptid-idl": "^0.3.0-alpha.1",
```

Comment:
I would have expected a cryptid dependency here. (e.g. in order to get the CryptidAccount information)